library/Rules/Unique.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+class Unique extends AbstractRule
+{
+    public function validate($input)
+    {
+        if (!is_array($input)) {
+            return false;
+        }
+        return count($input) == count(array_unique($input));
```

Comment:
`$input == array_unique($input)` is enough. Also, please, add a new line before `return`. 

***
As we [suggest](https://github.com/Respect/Validation/blob/master/CONTRIBUTING.md#coding-style-and-standards), run PHP Coding Standards Fixer to be sure.